### DISCUSSION TITLE
Multi-language managed properties for filters are not used in the correct language, only in the site default language.

### DISCUSSION THREAD

**Discussion by DanielMisch** [microsoft-search/pnp-modern-search#2859] [Q&A]
**Version used**
4.8

**Describe the bug**
If you have multilanguage managed properties and u using them in the filter webpart, only the default site language will be used to show the filters.

**To Reproduce**
- Site Language = 1033
- User/browser language = 1033
- filters shown in 1033

- Site Language = 1033
- User/browser language = 1031
- filters shown in 1033

- Site Language = 1031
- User/browser language = 1033
- filters shown in 1031

- Site Language = 1031
- User/browser language = 1031
- filters shown in 1031

**Expected behavior**
The user or browser choosen language should also applied to filter properties as they existing in the termstore.

**Screenshots**
--

**Desktop (please complete the following information):**
 - Browser chrome, edge most current version

**Additional context**
--

**Comment by wobba** [comment]
This is how indexing works and hasn't changed the past 10 years. If you use taxonomy fields you can make it work in the PnP web parts check `Enable localization` and https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/sharepoint-search/

**Reply by EbSe** [reply]
How does this work for the filter webpart? there is no Enable localization flag?!

**Reply by wobba** [reply]
The filter webpart relies on the setting from the main search webpart.

**Reply by EbSe** [reply]
Hmm still not working.

- Multilanguage Page (German and English) - german is default
- Browser set to English
- User is on english search results page
- Enable locaization is enabled on the search results webpart
- All webpart controls are translated into english
- Terms are translated into german and english (english is default)
- The values of the Managed Metadata field used in the filter webpart are still shown in german

any further suggestions?

Thanks in advance!!!

**Reply by wobba** [reply]
@EbSe see.. you are now adding more information on the setup, and you should log your own issue as it's different from the original question. As for translated pages, I see all OOB controls are also in English (following main UX language) on a translated page I have, and this is on SPFx. Not the web parts. 

The translation option in the web part will translate labels according to the UX language if applicable. Might be an issue for translated pages and filters?, but it should work in the main web part for translated pages last I tested. So please log a separate issue with full details. Has it worked in previous versions?

Code is at https://github.com/microsoft-search/pnp-modern-search/blob/main/search-parts/src/dataSources/SharePointSearchDataSource.ts#L1029C2-L1029C2 See line 202 where it picks locale from the translated URL.

**Reply by EbSe** [reply]
I will create a separate issue. Thanks
